apm-sniffer/apm-toolkit-activation/apm-toolkit-trace-activation/src/test/java/org/apache/skywalking/apm/toolkit/activation/trace/TagAnnotationTest.java

```diff
@@ -138,6 +144,145 @@ public void testTraceWithTags() throws Throwable {
 
     }
 
+    @Test
+    public void testTraceWithList() throws Throwable {
```

Comment:
```suggestion
    public void testTraceWithReturnList() throws Throwable {
```
I think the test method names are not accurate? Please confirm. Sam as other methods.